compiler-rt/lib/scudo/standalone/secondary.h

```diff
@@ -72,6 +72,7 @@ namespace {
 struct CachedBlock {
   static constexpr u16 CacheIndexMax = UINT16_MAX;
   static constexpr u16 InvalidEntry = CacheIndexMax;
+  static constexpr uptr ReleaseMemoryUpperBound = 1 << 14;
```

Comment:
Add a comment about why this value was chosen.

Reply:
Moved comment from retrieve code to here.